### DISCUSSION TITLE
How can i make this work on android?

### DISCUSSION THREAD

**Discussion by leoyanis** [lennart-finke/GodoTeX#13] [Q&A]
Hello there,
I was wondering how i could make this work in android or if there is any other way of displying latex in android like this. 
Anything helps, looking forward to your response.
:)

**Comment by lennart-finke** [comment]
Hello and thanks for your interest in GodoTeX! I've tried exporting to Android when first starting to develop and didn't have much success with the dependencies. However, I do think it should be possible in principle. Sadly I don't have time to work on this at the moment...

For further discussion, I suggest using the existing thread on the same topic: https://github.com/file-acomplaint/GodoTeX/discussions/5

**Reply by leoyanis** [reply]
Thanks for replying.
I have a question though. Right now i cant use the latex plugin on android because it doesnt load the plugin at all. On mac it works just fine just android not.
So i did some digging and i saw that the libSkiaSharp something is missing.

I tried including SkiaSharp native assets android but that doesnt work at all because i need .net 6.0 and android build tools 30.
But even when i do adjust everything to that version it doesnt let me.

Any ideas?
Thanks in advance
